File: UI/src/main/java/ca/warp7/rt/java/vc/VCFeature.java
package ca.warp7.rt.java.vc;

import ca.warp7.rt.java.core.ft.Feature;
import ca.warp7.rt.java.core.ft.FeatureItemTab;
import javafx.collections.FXCollections;
import javafx.collections.ObservableList;

public class VCFeature implements Feature {

    @Override
    public ObservableList<FeatureItemTab> getTabObservable() {
        return FXCollections.singletonObservableList(new FeatureItemTab(
                "Verification Center", "fas-check:16:gray", getFeatureId(),
                FeatureItemTab.Group.SingleTab, ""
        ));
    }

    @Override
    public String getFeatureId() {
        return "vc";
    }
}

File: UI/src/main/kotlin/ca/warp7/rt/ext/vc/VCFeature.kt
package ca.warp7.rt.ext.vc

import ca.warp7.rt.core.feature.Feature
import ca.warp7.rt.core.feature.FeatureItemTab
import ca.warp7.rt.core.feature.FeatureLink

class VCFeature : Feature {

    override val link = FeatureLink("Verification Center", "fas-check", 16)

    override val loadedTabs
        get() = listOf(FeatureItemTab(
                "Verification Center", "fas-check:16:gray", featureId,
                FeatureItemTab.Group.SingleTab, ""
        ))

    override val featureId get() = "vc"
}
